src/components/dialogs/network-modifications/two-windings-transformer/creation/two-windings-transformer-creation-dialog.js

```diff
@@ -444,9 +444,15 @@ const TwoWindingsTransformerCreationDialog = ({
                         twt?.[PHASE_TAP_CHANGER]
                     ),
                     currentLimiterRegulatingValue:
-                        twt?.[PHASE_TAP_CHANGER]?.regulationValue,
+                        twt?.[PHASE_TAP_CHANGER]?.[REGULATION_MODE] ===
+                        PHASE_REGULATION_MODES.CURRENT_LIMITER.id
+                            ? twt?.[PHASE_TAP_CHANGER]?.regulationValue
+                            : undefined,
```

Comment:
Maybe null instead of undefined as it is the default values in the method definition

Reply:
I wondered this too. Sometimes undefined is used (mostly for form default values), sometimes null is (especially for structures default values). I finally picked 'undefined' because that was the case in two-windings-transformer-modification-dialog but personnaly I prefer null.

Maybe a techlead should decide.